UPGRADE-2.1.md

```diff
@@ -0,0 +1,137 @@
+# UPGRADE FROM 2.0 TO 2.1
+
+**Table of Contents:**
+
+- [Disclaimer](#disclaimer)
+- [Migrate your standard project](#migrate-your-standard-project)
+- [Migrate your custom code](#migrate-your-custom-code)
+
+## Disclaimer
+
+> Please check that you're using Akeneo PIM v2.0
+
+> We're assuming that you created your project from the standard distribution
+
+> This documentation helps to migrate projects based on the Community Edition
+
+> Please perform a backup of your database before proceeding to the migration. You can use tools like [mysqldump](https://dev.mysql.com/doc/refman/5.7/en/mysqldump.html).
+
+> Please perform a backup of your codebase if you don't use a VCS (Version Control System).
+
+## Migrate your standard project
+
+1. Download it from the website [PIM community standard](http://www.akeneo.com/download/) and extract:
+
+    ```bash
+    wget http://download.akeneo.com/pim-community-standard-v1.7-latest.tar.gz
+    tar -zxf pim-community-standard-v1.7-latest.tar.gz
+    cd pim-community-standard/
+    ```
+
+2. Copy the following files to your PIM installation:
+
+    ```bash
+    export PIM_DIR=/path/to/your/pim/installation
+ 
+    mv $PIM_DIR/app/config/config.yml $PIM_DIR/app/config/config.yml.bak
+    cp app/config/config.yml $PIM_DIR/app/config
+ 
+    mv $PIM_DIR/app/config/pim_parameters.yml $PIM_DIR/app/config/pim_parameters.yml.bak
+    cp app/config/pim_parameters.yml $PIM_DIR/app/config
+
+    mv $PIM_DIR/composer.json $PIM_DIR/composer.json.bak
+    cp composer.json $PIM_DIR/
+    ```
+
+3. Remove your old upgrades folder:
+
+    ```bash
+    rm -rf $PIM_DIR/upgrades/schema
+    ```
+
+4. Update your dependencies:
+
+    * [Optional] If you had added dependencies to your project, you will need to do it again in your `composer.json`.
+    You can display the differences of your previous composer.json in `$PIM_DIR/composer.json.bak`.
+    
+    ```JSON
+    "require": {
+       "your/dependency": "version",
+       "your/other-dependency": "version",
+    }
+    ```
+
+   * Then run the command to update your dependencies:
+
+    ```bash
+    cd $PIM_DIR
+    php -d memory_limit=3G ../composer.phar update
+    ```
+    
+    This step will copy the upgrades folder from `pim-community-dev/` to your Pim project root in order to migrate.
+    If you have custom code in your project, this step may raise errors in the "post-script" command.
+    In this case, go to the chapter "Migrate your custom code" before running the database migration.
+ 
+5. Migrate your database:
+ 
+    ```bash
+    rm -rf var/cache
+    bin/console doctrine:migration:migrate --env=prod
+    ```
+
+6. Create the missing internal job instances:
+
+    Depending on the version of your current PIM, you'll have to execute these commands to create new internal jobs.
+
+    * From 2.0.4 and below
```

Comment:
You should explain here that if you want to migrate to the PIM 2.1 you should/must (?) be on the last patch of the 2.0. It will avoid those lines. Not your fault, but this type of upgrade like : "if you upgrade 2.0.x do this, if you you upgrade from 2.0.y do this" should never exist.
Wait comments from other team members before taking into account my comment.

Reply:
It's an alternative of what I said and I'm wondering if your solution is better or not.


But something will be mandatory: adding jobs in patch (😞 ) of the 2.0 imply to update the changelog of the 2.1 (for example, a patch wih a job in 2.0.11, and then merge this patch in 2.1).